ts/components/animations/ShakeAnimation.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import { Animated, Easing } from "react-native";
+
+type Props = Readonly<{}>;
+/**
+ *
+ * This component create a custom animation called 'shake'
+ */
+export class ShakeAnimation extends React.PureComponent<Props> {
```

Comment:
as it doesn't have any state nor any lifecycle hook, I guess this can be a stateless functional component (React.SFC) - what do you think @fpersico ?

Reply:
ah but you need to expose the `shake` method, not sure SFC supports that